docker-compose.yml

```diff
@@ -44,6 +44,9 @@ services:
       # we can put that name inside our integration test certs (e.g. as a crl
       # url) and have it look like a publicly-accessible name.
       - "ca.example.org:10.77.77.77"
+      # Allow the boulder container to be reached as "integration.trust", for
+      # similar reasons, but intended for use as a SAN rather than a CRLDP.
+      - "integration.trust:10.77.77.77"
```

Comment:
.trust is a real TLD which has some interesting properties (the entire TLD is HSTS preloaded).  It might be better to pick a different test domain, even if it's OK right now.

Reply:
It _has_ to be a real TLD, or we'll refuse to issue for it. I picked this one because it sounded good and wikipedia indicated that no one was actually allowing registrations under it. Perhaps I shouldn't have trusted wikipedia for that.

Happy to take other suggestions, but it does have to be a real TLD.

Reply:
NCC group definitely at least used to use it for a bunch of stuff.
Feels more-than-usually-likely for generic TLDs to go away too, so even more reason to not use it if they've stopped allowing registrations
I'd probably use something under example.org instead.

Reply:
Can't use anything under example.org, we forbid issuance for example.org :)

Reply:
hah well, it's honestly probably fine :)